### DISCUSSION TITLE
How to copy local `pb_data` and `migrations` to an already deployed instance on fly.io?

### DISCUSSION THREAD

**Discussion by shyakadavis** [pocketbase/pocketbase#4217] [Q&A] [answered]
Hi;

Having successfully completed https://github.com/pocketbase/pocketbase/discussions/537 (thanks, @ganigeorgiev), I was curious about this;

Say I have modified my local collections + api rules, and didn't want to go through the trouble of manually re-doing everything on my deployed version of PB on fly.io, (error-prone e.t.c) how could I apply my local migrations, api rules, and data (optional) to the deployed instance?

Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
There is no builtin mechanism to copy data from one instance to another.

For the collections configuration (eg. new fields, api rules changes, etc.) as long as they are part of a migration you can copy the `pb_migrations` dir in your Dockerfile (in the example Dockerfile you can uncomment the pb_hooks and pb_migrations `COPY` instructions) and the unapplied changes from there will be applied automatically on redeploy (aka. rerun `flyctl deploy`).
